service/src/utils/onchain-data.ts

```diff
@@ -122,7 +131,7 @@ export async function unwrapSarcophagus(web3Interface: Web3Interface, sarcoId: s
   try {
     await web3Interface.archaeologistFacet.unwrapSarcophagus(
       sarcoId,
-      ethers.utils.arrayify(decryptedShard) // TODO: Might need an extra `Buffer.from`. `unwrapSarcophagus` expects a solidity bytes type, `decryptedShard` is a hex string
+      decryptedShard
```

Comment:
Interesting. The '0x`- prefixed hex string is treated as `bytes` in the contract?

Reply:
Very good to know